### PR TITLE
Added Reference URL

### PR DESCRIPTION
**Pull request by rubaljain**

### CODE REVIEW

File: Testing for APIs.md [hunk 1]
```diff
@@ -117,6 +117,13 @@ This will send to the server a request with the session ID or token that was inv
 * [Privilege escalation](https://github.com/OWASP/OWASP-Testing-Guide-v5/blob/master/document/4%20Web%20Application%20Security%20Testing/4.6%20Authorization%20Testing/4.6.3%20Testing%20for%20Privilege%20escalation%20(OTG-AUTHZ-003).md)
 * All [Session Management](https://github.com/OWASP/OWASP-Testing-Guide-v5/tree/master/document/4%20Web%20Application%20Security%20Testing/4.7%20Session%20Management%20Testing) test cases
 
+# Brute Force Weak Secrets used for JWT
+Step 1: Clone the repository https://github.com/brendan-rius/c-jwt-cracker
+Step 2: Navigate to cloned repository and compile the source code using 'make'
+Step 3: Run ./jwtcrack <JWT>
+
+In case of weak secret value, this will bruteforce the secret.
+
```
Comment (kingthorin) [inline Testing for APIs.md]: Maybe this section should be a level 2 section (move it up to L114?). Similar to "Specific Testing – Testing (Token-Based) Authentication"

Comment (rubaljain) [reply Testing for APIs.md]: Moved

Comment (kingthorin) [reply Testing for APIs.md]: Thanks, closing this PR since you opened another one.

**Comment by kingthorin** [comment]
Closing, see #45 .

**Pull request closed without merging**